README.md

```diff
@@ -2,6 +2,20 @@
 
 The DHL API Javascript client implementation.
 
+```javascript
+const dhlAPI = require("dhl-api");
+
+// instance the DHL client with defaults
```

Comment:
better "// instance the API client with defaults" to keep parity with the UPS one